modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc

```diff
@@ -6,32 +6,36 @@
 [id='ipi-install-firmware-requirements-for-installing-with-virtual-media_{context}']
 = Firmware requirements for installing with virtual media
 
-The installer for installer-provisioned {product-title} clusters validates the hardware and firmware compatibility with Redfish virtual media. The following table lists the minimum firmware versions tested and verified to work for installer-provisioned {product-title} clusters deployed by using Redfish virtual media.
+The installation program for installer-provisioned {product-title} clusters validates the hardware and firmware compatibility with Redfish virtual media. The installation program will not begin installation on a node if the node firmware is not compatible. The following tables list the minimum firmware versions tested and verified to work for installer-provisioned {product-title} clusters deployed by using Redfish virtual media.
 
-.Firmware compatibility for Redfish virtual media
+[NOTE]
+====
+Red Hat does not test every combination of firmware, hardware, or other third-party components. For further information about third-party support, see link:https://access.redhat.com/third-party-software-support[Red Hat third-party support policy]. See the hardware documentation for the nodes or contact the hardware vendor for information about updating the firmware.
```

Comment:
For parallelism throughout the note, you could reverse the structure of the last sentence as follows:

> For information about updating the firmware, see the hardware documentation for the nodes or contact the hardware vendor.